com.sweethome3d.Desktop.appdata.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally it should be contributed upstream.

Reply:
I'd rather have it reviewed first. As it is, there isn't even a .desktop file, so I'd rather propose everything upstream once I've got something to show for it.